lib/remote/modifyobjecthandler.cpp

```diff
@@ -87,23 +97,29 @@ bool ModifyObjectHandler::HandleRequest(
 
 		String key;
 
-		try {
-			if (attrs) {
-				ObjectLock olock(attrs);
-				for (const Dictionary::Pair& kv : attrs) {
-					key = kv.first;
-					obj->ModifyAttribute(kv.first, kv.second);
+		if (obj->GetPackage() != package) {
```

Comment:
Hi @julianbrost , @Al2Klimov 
we found the following problem with this check:
- a host is created via the API in the package "director" 
- then it is edited via the icingaweb2 page (Overview -> Hosts -> Feature Commands) 
- the change is not applied because the code enters in this if

One possible solution is to validate the package only when the package parameter is specified in the API request.
Does this make sense?

Reply:
Sounds like a plan.